### PR TITLE
#154799897 Create incident timeline page

### PR DESCRIPTION
**Pull request by NaiRobley**
#### What does this PR do?
Create the incident timeline page

#### Description of Task to be completed?
As user, WHEN I click on an incident subject,
THEN I should see the incident timeline page as designed in the mockups.

#### Where should the reviewer start?
[Timeline Component](https://github.com/AndelaOSP/wire/tree/ft-incident-timeline-154799897/src/pages/IncidentTimeline)

#### What are the relevant pivotal tracker stories?
[#154799897](https://www.pivotaltracker.com/story/show/154799897)

#### Screenshots (if appropriate)
image

image

### CODE REVIEW

**Comment by Mnickii** [comment]
The logo, the search box, the notification icon and the activity cards have deviated from the mockup design

**Comment by RayNjeri** [comment]
screen shot 2018-02-13 at 2 10 36 pm
I have a concern here... how would a user know that these bits here are drop downs one can change the incidents status and the assignee??

**Comment by NaiRobley** [comment]
@RayNjeri that part needs new colors as the dropdown bottom line and icon is almost same color as the background

**Comment by NaiRobley** [comment]
@RayNjeri Updated to black
screen shot 2018-02-13 at 14 27 42

File: src/pages/IncidentTimeline/IncidentTimeline.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  .timeline-main-content {
+    justify-content: flex-end;
+    width: 100vw;
+    padding: 0 1rem;
+    z-index: 0;
+    position: relative;
+
+      .chat-tab {
```
Comment (Mbarak-Mbigo) [inline src/pages/IncidentTimeline/IncidentTimeline.scss]: Why do we have `classes defined with no styling?` If the styling is to be added later, kindly provide a comment or any form of directive on these classes.

Comment (NaiRobley) [reply src/pages/IncidentTimeline/IncidentTimeline.scss]: On it

**Comment by NaiRobley** [comment]
@RayNjeri The names will be sent along with the incident

File: src/pages/IncidentTimeline/IncidentTimeline.scss (hunk 1 above)
Comment (NaiRobley) [reply src/pages/IncidentTimeline/IncidentTimeline.scss]: fixed

**Comment by NaiRobley** [comment]
@Mbarak-Mbigo yes

**Comment by NaiRobley** [comment]
@Mnickii The notifications have been fixed, awaiting logo from design guys but that can be changed anytime

**Pull request merged**